tests/test_oidc.py

```diff
@@ -91,3 +93,38 @@ def it_is_possible_to_get_iat(self, mock_auth, client, db_session):
         assert response.status_code == 302
 
         assert session['iat'] == expected_issue_timestamp
+
+
+def mock_requests_get(*args, **kwargs):
+    class MockResponse:
+
+        def __init__(self, json_data, status_code):
+            self.json_data = json_data
+            self.status_code = status_code
+
+        def json(self):
+            return self.json_data
+
+    if 'https://login.microsoftonline.com/testid' in args[0]:
+        return MockResponse({
+            'authorization_endpoint':
+            'https://login.microsoftonline.com/testid/oauth2/authorize',
+            'discovery_url':
+            'https://login.microsoftonline.com/testid'}, 200)
```

Comment:
The status code is easily overlooked when formatted like this, maybe something like this would be more readable:
```python
json = {'authorization_endpoint': 'https://...'}
return MockResponse(json, 200)
```